src/kits/arm/arm_node.cpp

```diff
@@ -141,14 +160,23 @@ class ArmNode {
   }
 
   void startArmMotion(const hebi_cpp_api_examples::ArmMotionGoalConstPtr& goal) {
-    ROS_INFO("Executing arm motion action");
+    auto last_position = arm_.lastFeedback().getPosition();
+    //ROS_INFO("Executing arm motion action");
+    ROS_INFO_STREAM("Starting From Joint Position: " << last_position[0] << " | "
+                                                     << last_position[1] << " | "
+                                                     << last_position[2] << " | "
+                                                     << last_position[3] << " | "
+                                                     << last_position[4] << " | "
+                                                     << last_position[5]);
```

Comment:
I would remove the debugging code here and lines 178-179

Reply:
Fixed